packages/bpk-stylesheets/webpack.config.js

```diff
@@ -106,6 +114,10 @@ module.exports = {
   },
 
   plugins: [
+    new WrapperPlugin({
+      test: /\.css$/,
+      header: license,
```

Comment:
Would it be possible to reuse the text blob in [license-header.js](https://github.com/Skyscanner/backpack/blob/master/packages/bpk-tokens/formatters/license-header.js) instead of introducing another place were the comment is present?